io.github.jliljebl.Flowblade.yaml

```diff
@@ -247,30 +204,42 @@ modules:
         url: https://www.ffmpeg.org/releases/ffmpeg-4.2.1.tar.xz
         sha256: cec7c87e9b60d174509e263ac4011b522385fd0775292e1670ecc1180c9bb6d4
 
+  - name: numpy
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/python3.7/site-packages
+      - CFLAGS='-L/usr/lib -Lbuild/temp.linux-x86_64-3.4 -I/usr/include -I/usr/include/python3.7m/' CXX=/usr/bin/g++ CC=/usr/bin/gcc PYTHONUSERBASE=/app/ python3 setup.py install --prefix=/app
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/ff/59/d3f6d46aa1fd220d020bdd61e76ca51f6548c6ad6d24ddb614f4037cf49d/numpy-1.17.4.zip
+        sha256: f58913e9227400f1395c7b800503ebfdb0772f1c33ff8cb4d6451c06cabdf316
+
   - name: mlt
     config-opts:
       - --enable-gpl
       - --enable-gpl3
       - --swig-languages=python
     build-options:
       env:
-        PYTHON: python2
+        PYTHON: python3
         CXXFLAGS: -L/app/lib
     post-install:
-      - install -Dm644 src/swig/python/mlt.py /app/lib/python2.7/site-packages/mlt.py
-      - install src/swig/python/_mlt.so /app/lib/python2.7/site-packages/_mlt.so
+      - install -Dm644 src/swig/python/mlt.py /app/lib/python3.7/site-packages/mlt.py
+      - install src/swig/python/_mlt.so /app/lib/python3.7/site-packages/_mlt.so
     cleanup:
       - /bin
     sources:
       - type: git
         url: https://github.com/mltframework/mlt.git
-        commit: 1191b521b741dddb92d008a31e375077b337c11d
+        commit: 221ff239b9cbbc0e0b749f24e3f754eb087cb1f9
 
   - name: flowblade
     buildsystem: simple
     subdir: flowblade-trunk
     build-commands:
-      - python2 setup.py install --prefix=/app --install-lib=/app/share/flowblade
+      - python3 setup.py install --prefix=/app --install-lib=/app/share/flowblade
     post-install:
       - desktop-file-edit --set-key=Exec --set-value='flowblade %f' /app/share/applications/io.github.jliljebl.Flowblade.desktop
       - sed -i 's|"app|"/app|' /app/share/flowblade/Flowblade/translations.py
```

Comment:
You can drop that line, since this was fixed upstream (https://github.com/jliljebl/flowblade/pull/598).

Reply:
Ok, Ill push update that drops:
 - sed -i 's|"app|"/app|' /app/share/flowblade/Flowblade/translations.py